ui/app/helpers/utils/metametrics.util.js

```diff
@@ -4,6 +4,8 @@ import ethUtil from 'ethereumjs-util'
 
 const inDevelopment = process.env.METAMASK_DEBUG || process.env.IN_TEST
 
+export const mixpanelId = inDevelopment ? 1 : process.env.MIXPANEL_ID ?? 2
```

Comment:
Do these IDs correspond to anything? :thinking: They did for Matomo, but I suspect they don't for Segment. I'd suggest omitting them unless there is a "development" project on Segment we want to point at, that we don't mind making the ID easily available for.